specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2018-10-10/table.json

```diff
@@ -0,0 +1,1585 @@
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageServiceStats"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageError": {
+      "type": "object",
+      "properties": {
+        "Message": {
+          "type": "string"
+        }
+      }
+    },
+    "StorageServiceProperties": {
+      "description": "Storage Service Properties.",
+      "type": "object",
+      "properties": {
+        "Logging": {
+          "description": "Azure Analytics Logging settings",
+          "$ref": "#/definitions/Logging"
+        },
+        "HourMetrics": {
+          "description": "A summary of request statistics grouped by API in hourly aggregates for queues",
+          "$ref": "#/definitions/Metrics"
+        },
+        "MinuteMetrics": {
+          "description": "a summary of request statistics grouped by API in minute aggregates for queues",
+          "$ref": "#/definitions/Metrics"
+        },
+        "Cors": {
+          "description": "The set of CORS rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CorsRule",
+            "xml": {
+              "name": "CorsRule"
+            }
+          },
+          "xml": {
+            "wrapped": true
+          }
+        }
+      }
+    },
+    "Logging": {
+      "description": "Azure Analytics Logging settings.",
+      "type": "object",
+      "required": [
+        "Version",
+        "Delete",
+        "Read",
+        "Write",
+        "RetentionPolicy"
+      ],
+      "properties": {
+        "Version": {
+          "description": "The version of Storage Analytics to configure.",
+          "type": "string"
+        },
+        "Delete": {
+          "description": "Indicates whether all delete requests should be logged.",
+          "type": "boolean"
+        },
+        "Read": {
+          "description": "Indicates whether all read requests should be logged.",
+          "type": "boolean"
+        },
+        "Write": {
+          "description": "Indicates whether all write requests should be logged.",
+          "type": "boolean"
+        },
+        "RetentionPolicy": {
+          "$ref": "#/definitions/RetentionPolicy"
+        }
+      }
+    },
+    "Metrics": {
```

Comment:
This generates `Error AZC0012 Single word class names are too generic and have high chance of collision with BCL types or types from other libraries`

Could we rename it to `RequestMetrics`?